Scripts/Tracking/Follow/ObjectFollow.cs

```diff
@@ -0,0 +1,217 @@
+﻿namespace VRTK.Core.Tracking.Follow
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using System;
+    using VRTK.Core.Data.Attribute;
+    using VRTK.Core.Data.Enum;
+    using VRTK.Core.Process;
+    using VRTK.Core.Tracking.Follow.Modifier;
+
+    /// <summary>
+    /// The ObjectFollow mirrors the transform properties of another object based on the given FollowModifier.
+    /// </summary>
+    public class ObjectFollow : SourceTargetProcessor
+    {
+        [Header("Object Follow Settings")]
+
+        [UnityFlag]
+        [Tooltip("The transform properties to follow.")]
+        public TransformProperties follow;

```

Comment:
Shall we specify a default value for value-types to ensure us changing the enum values in the type itself doesn't change the default of the usages?

Reply:
you mean do like 

`public TransformProperties follow = TransformProperties.Position | TransformProperties.Rotation | TransformProperties.Scale;`